docs/DeveloperGuide.md

```diff
@@ -5,13 +5,31 @@ title: Developer Guide
 
 ## **Introduction**
 
-### Purpose
+Welcome! We are excited that you want to learn about <strong><i>TBM</i></strong>.<br>
+This page contains step-by-step instructions on how to get started in contributing or getting yourself involved in this awesome project! However, before you get too excited, allow us to introduce you to the core concepts that underpins the heart of <strong><i>TBM</i></strong>.
```

Comment:
I think for `<strong><i>` we should follow the markdown conventions and use `**` and `_`?

According to https://se-education.org/guides/conventions/markdown.html

Reply:
Oh nice, I keep thinking markdown has no italics.